_schemaV2.graphql

```diff
@@ -2120,6 +2120,12 @@ union ArtworkMutationType =
 
 union ArtworkOrEditionSetType = Artwork | EditionSet
 
+type ArtworkPriceInsights {
+  artistId: String
```

Comment:
**(question):** Why do we return the artist ID?

Reply:
Good question actually, I only added it because it is used in the query, would it be okay to remove it?
because I don't think we need this information inside the ArtworkPriceInsights